sql-cloudant/README.md

```diff
@@ -31,15 +31,14 @@ The `--packages` argument can also be used with `bin/spark-submit`.
 
 Submit a job in Python:
     
-    spark-submit  --master local[4] --jars <path to cloudant-spark.jar>  <path to python script> 
+    spark-submit  --master local[4] --packages org.apache.bahir:spark-sql-cloudant_2.11:2.2.0-SNAPSHOT  <path to python script> 
```

Comment:
Good point, this should probably be something like `--packages org.apache.bahir:spark-sql-cloudant_SCALA_VERSION:PACKAGE_VERSION`